streamnative/README.md

```diff
@@ -0,0 +1,73 @@
+# StreamNative
+
+## Overview
+
+[StreamNative][1] provides an enterprise-grade messaging and event streaming platform built on Apache Pulsar. It offers scalable, real-time data streaming solutions with features like multi-tenancy, geo-replication, and seamless integration with cloud services.
+
+The StreamNative integration collects below types of [metrics][2]:
+
+1. Health
+2. Pulsar Resource
+3. Source Connector
+4. Sink Connector
+5. Kafka Connect
+
+## Setup
+
+### Configuration
+
+#### Get StreamNative Credentials
+
+1. Login to the [StreamNative Cloud Console Account][3].
+2. Get the `Organization ID` and `Instance Name`
```

Comment:
```suggestion
2. Get the `Organization ID` and `Instance Name`:
```

Reply:
You might consider making these top-level numbered items H5s instead, so the list won't be quite so long. I also recommend using a numbered list when the items need to be completed in order, rather than the bullets used below.

Reply:
Done.